dotcom-rendering/src/web/components/MatchStats.tsx

```diff
@@ -173,19 +256,95 @@ const H4 = ({ children }: { children: React.ReactNode }) => (
 	</h4>
 );
 
-export const MatchStats = ({ home, away }: Props) => (
+const DecideDoughnut = ({
+	home,
+	away,
+	format,
+}: {
+	home: TeamType;
+	away: TeamType;
+	format: ArticleFormat;
+}) => {
+	const sections = [
+		{
+			value: home.possession,
+			label: home.codename,
+			color: home.colours,
+		},
+		{
+			value: away.possession,
+			label: away.codename,
+			color: away.colours,
+		},
+	].reverse();
```

Comment:
```suggestion
	const sections = [
		{
			value: away.possession,
			label: away.codename,
			color: away.colours,
		},
		{
			value: home.possession,
			label: home.codename,
			color: home.colours,
		},
	];
```

Could we change the default ordering so we don't have to `reverse()`? Could be going mad so feel free to ignore!

Reply:
I don't fully understand why we have to do that in the first place.....